src/takedown.js

```diff
@@ -0,0 +1,85 @@
+import { getRepliesInBatch, getUserReplies } from './util/graphql.js';
+import { getSpamList } from './getSpamList.js';
+import { getAllPRs, createPullRequest } from './githubPR.js';
+import subTime from 'date-fns/sub';
+
+const knownUsers = new Set();
+
+async function initKnownUsers() {
+  const list = await getAllPRs();
+  list.forEach((pr) => {
+    // get userid from lastWord of pr title
+    const title = pr.title;
+    if (title.startsWith('Takedown spam user')) {
+      const userid = pr.title.slice(pr.title.lastIndexOf(' ') + 1);
+      // console.log('userid: ', userid);
+      if (!knownUsers.has(userid)) knownUsers.add(userid);
+    }
+  });
+}
+
+async function getSpamRepliesFromDate(date) {
```

Comment:
nit: since this method actually have side effects (create pull requests) we should name the function to something like `processSpamRepliesFromDate`. Functions starting with `get` sounds like a pure function that should not do such things.